pubnub-users/src/integrationTest/kotlin/com/pubnub/entities/UserIntegTest.kt

```diff
@@ -23,13 +24,18 @@ class UserIntegTest() {
     private val EXTERNAL_ID = "externalId"
     private val PROFILE_URL = "profileUrl"
     private val EMAIL = "email"
-    private val CUSTOM: Map<String, Any> = mapOf("favouriteNumber" to 1, "favouriteColour" to "green")
+    private val CUSTOM = mapOf("favouriteNumber" to 1, "favouriteColour" to "green")
+    private val TYPE = "type"
+    private val STATUS = "status"
 
     @BeforeEach
     fun setUp() {
         val config = PNConfiguration("kotlin").apply {
-            subscribeKey = IntegTestConf.subscribeKey
+            subscribeKey = "sub-c-645d1439-e412-4123-be12-265bda8711a5"
             publishKey = IntegTestConf.publishKey
+            origin = "ingress-tcp-pub.pdx1.aws.int.ps.pn"
```

Comment:
Maybe we could extract origin value to IntegrationTestConfiguration?

Reply:
yeah. Sure. Why not